internal/telemetry/event_test.go

```diff
@@ -329,6 +329,33 @@ func TestWithExtraProps(t *testing.T) {
 	a.Equal(true, e.Properties["a"])
 }
 
+func TestSanitizePrompt(t *testing.T) {
```

Comment:
Those are just default settings that are displayed at the start - there is no user interaction and nothing that we are required to track in that regard.

Reply:
@blva would you link the code?

Reply:
https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/atlas/quickstart/confirm_default_setup.go

Reply:
looking at the code I think it's not tracked, so I think we can resolve this!

Reply:
(my previous comment mysteriously appeared in two places, and when I removed it from the other place it disappeared altogether...)

@blva the suggestion is just output to stdout - it doesn't get tracked because it's not part of an actual prompt, ie. you won't find any of that output in any track event.